packages/info/lib/logger.js

```diff
@@ -1,19 +1,9 @@
 'use strict';
 /* eslint-disable no-console */
 
-const { EOL } = require('os');
-
 const chalk = require('chalk');
-const wordWrap = require('word-wrap');
```

Comment:
I don't see an updated package.json in this PR - shouldn't we remove this dependency?